### PR TITLE
adds oxygen to tower caps

### PR DESCRIPTION
**Pull request by boodaliboo**
its a tree kinda
#### Changelog

:cl:  
rscadd: Adds oxygen to tower caps
/:cl:

### CODE REVIEW

**Comment by Moja10** [comment]
When botany mains make PRs like this, the first thing that should be done is looking up every chem that uses oxygen as part of a recipe and see what chems can now be made when combined with already existing chem from other plants. Then you will know their true intentions.

File: code/modules/hydroponics/grown/towercap.dm [hunk 1]
```diff
@@ -16,6 +16,8 @@
 	icon_dead = "towercap-dead"
 	genes = list(/datum/plant_gene/trait/plant_type/fungal_metabolism)
 	mutatelist = list(/obj/item/seeds/tower/steel)
+	reagents_add = list(/datum/reagent/oxygen = 0.1)
+
```
Comment (JamieD1) [inline code/modules/hydroponics/grown/towercap.dm:20]: Can you remove the extra random line?

**Review by JamieD1 — CHANGES_REQUESTED** [review]

**Comment by SomeguyManperson** [comment]
importantly this allows botanists to fabricate their own saltpetre since nitrogen is somewhat ample in supply from canned air

**Comment by alexkar598** [comment]
Id rather not powercreepe botany any further.

**Pull request closed without merging**